src/sentry/grouping/strategies/message.py

```diff
@@ -260,8 +260,8 @@ def _handle_match(match: Match[str]) -> str:
                 # Register 100 analytics events per experiment per instance restart
                 # This generates samples for review consistently but creates a hard cap on
                 # analytics event volume
-                experiment.counter += 1
-                if experiment.counter < 100:
+                if share_analytics and experiment.counter < 100:
```

Comment:
@mikejihbe This counter thing seem weird to me. Each instance of Sentry has it's own localized counter for each experiment? Seems like it would have been cleaner to just do sampling here record it 1% of the time instead of adding this state.

Reply:
Even 1% is too many. I just need some samples to look at to validate.

Reply:
pick a different number then?